src/bindgen.rs

```diff
@@ -1,24 +1,67 @@
 use emoji;
 use error::Error;
 use progressbar::Step;
-use std::process::Command;
+use std::{path, process::Command};
 use PBAR;
 
-pub fn cargo_install_wasm_bindgen(step: &Step) -> Result<(), Error> {
+#[cfg(not(target_family = "windows"))]
+static LOCAL_BINDGEN_PATH: &str = "bin/wasm-bindgen";
+
+#[cfg(target_family = "windows")]
+static LOCAL_BINDGEN_PATH: &str = "bin\\wasm-bindgen";
+
+fn local_wasm_bindgen_path_str(crate_path: &str) -> String {
+    #[cfg(not(target_family = "windows"))]
+    return format!("{}/{}", crate_path, LOCAL_BINDGEN_PATH);
+    #[cfg(target_family = "windows")]
+    return format!("{}\\{}", crate_path, LOCAL_BINDGEN_PATH);
+}
+
+pub fn wasm_bindgen_version_check(
+    crate_path: &str,
+    dep_version: &str,
+    step: &Step,
+) -> Result<bool, Error> {
+    let msg = format!("{}Checking WASM-bindgen dependency...", emoji::CHECK);
+    PBAR.step(step, &msg);
+
+    let wasm_bindgen = local_wasm_bindgen_path_str(crate_path);
+    if !path::Path::new(&wasm_bindgen).is_file() {
+        return Ok(false);
+    }
+
+    let output = Command::new(wasm_bindgen).arg("--version").output()?;
+    if output.status.success() {
+        let s = String::from_utf8_lossy(&output.stdout);
+        let installed_version = s.trim();
+        Ok(installed_version == dep_version)
+    } else {
+        let error_msg = "Could not find version of local wasm-bindgen";
```

Comment:
we may want to give some sort of instruction about how to handle this if it happens. feels a little like a dead end right now :)